components/Hits.js

```diff
@@ -1,11 +1,9 @@
 var React = require('react');
 var map = require('lodash/collection/map');
 
-var Template = require('./Template');
-
 class Hits extends React.Component {
   renderWithResults() {
-    var renderedHits = map(this.props.hits, (hit) => {
+    var renderedHits = map(this.props.results.hits, (hit) => {
```

Comment:
While writing the tests I wondered why we where splitting the results object in two (hits: results.hits, results: results). If we already are dealing with the whole `results` then we might just pass it and use it inside the component. Since it already knows about the whole `results` object.